GeoIP2/Http/SyncClient.cs

```diff
@@ -22,21 +24,29 @@ ProductInfoHeaderValue userAgent
             )
         {
             _auth = auth;
+#if !NETSTANDARD1_4
             _timeout = timeout;
             _userAgent = userAgent.ToString();
+#endif
         }
 
         public Response Get(Uri uri)
         {
             var request = (HttpWebRequest) WebRequest.Create(uri);
+#if !NETSTANDARD1_4
             request.Timeout = _timeout;
             request.UserAgent = _userAgent;
+#endif
             request.Headers["Authorization"] = $"Basic {_auth}";
 
             HttpWebResponse response;
             try
             {
+#if !NETSTANDARD1_4
                 response = (HttpWebResponse) request.GetResponse();
+#else
+                response = (HttpWebResponse) request.GetResponseAsync().Result;
```

Comment:
There isn't a synchronous method in netstandard1.4? This change is quite problematic as it doesn't wait for the task to complete, it doesn't handle the exceptions properly, and blocking on async code can lead to deadlocks.